### DISCUSSION TITLE
Display of the kWh saved by regenerative braking

### DISCUSSION THREAD

**Discussion by Tesalro** [teslamate-org/teslamate#2821] [Ideas]
Hi, 
Could it be possible to display the kWh saved by regenerative braking ?
I mean by trip or by day or by period of time
I don't know if the datas are available in API or if it can be calculated

By the way nice software :-)
Thanks

**Comment by lq313c** [comment]
It should be possible to integrate over time where power < 0. I'm not sure if that equates to power charged back into the battery though. I'm guessing there's some efficiency loss in the process?

**Reply by Tesalro** [reply]
I don't know :-(
Need to think about it
Does anybody have an idea how to proceed ?
